content/guides/models/app/features/panels/_index.md

```diff
@@ -61,6 +61,43 @@ To add a custom panel to your workspace:
 
 To learn more about the options for each type of panel, refer to the relevant section below, such as [Line plots]({{< relref "line-plot/" >}}) or [Bar plots]({{< relref "bar-plot.md" >}}).
 
+## Share panels
+
+This section describes the various ways to share a panel and how each way differs.
+
+The settings of a project determine who can view the panel. This means that if the project is private, only members of the project can view the panel. If the project is public, anyone with the link can view the panel.
+
+If multiple panels have the same name, W&B shares the first panel with the name.
```

Comment:
This sentence feels a little out of place here. maybe nest it under "View and share a panel in full-screen mode", or create a new section along the lines of "Create URL by panel name".

The format of the URL is:

```
https://wandb.ai/ENTITY_NAME/PROJECT_NAME?panelDisplayName=PANEL_NAME&panelSectionName=SECTON_NAME
```

Where `PANEL_NAME` is most commonly just the metric name, and the section name param is actually optional (but is one strategy for disambiguating if there are multiple panels with the same name across the workspace)

Knowing how to construct a panel URL is probably most relevant for those creating automations (e.g. user sets up a slack bot that will alert if a metric does something weird, and they want to include a link to the panel by the metric name)

Reply:
Done!